src/Microsoft.ML/Data/TextLoader.cs

```diff
@@ -184,11 +184,11 @@ private static bool TryGetDataKind(Type type, out DataKind kind)
                 kind = DataKind.TX;
             else if (type == typeof(DvBool) || type == typeof(bool))
                 kind = DataKind.BL;
-            else if (type == typeof(DvTimeSpan) || type == typeof(TimeSpan))
+            else if (type == typeof(TimeSpan))
                 kind = DataKind.TS;
-            else if (type == typeof(DvDateTime) || type == typeof(DateTime))
+            else if (type == typeof(DateTime))
                 kind = DataKind.DT;
-            else if (type == typeof(DvDateTimeZone) || type == typeof(TimeZoneInfo))
+            else if (type == typeof(DateTimeOffset) || type == typeof(TimeZoneInfo))
```

Comment:
>TimeZoneInfo [](start = 70, length = 12)

In what way does a `TimeZoneInfo` map into a `DateTimeOffset`?

Reply:
I agree this seems totally wrong.

---
In reply to: [211315779](https://github.com/dotnet/machinelearning/pull/693#discussion_r211315779) [](ancestors = 211315779)

Reply:
It was a miss, I did intend to remove it.

---
In reply to: [211662608](https://github.com/dotnet/machinelearning/pull/693#discussion_r211662608) [](ancestors = 211662608,211315779)